bindings/pydeck/pydeck/io/html.py

```diff
@@ -10,6 +11,16 @@
 from ..frontend_semver import DECKGL_SEMVER
 
 
+def in_jupyter():
+    try:
+        ip = get_ipython()  # noqa
+
+        if ip.has_trait("kernel"):
```

Comment:
This function will return `None` if `ip.has_trait("kernel")` is false. Maybe just do `return ip.has_trait("kernel")`?